packages/cli/src/build/build.ts

```diff
@@ -216,6 +211,22 @@ const buildAndOutputNonComponentFiles = async (targetContext: TargetContextWithC
   return await outputNonComponentFiles({ ...targetContext, files });
 };
 
+function runHook(hook: string, config?: MitosisConfig) {
```

Comment:
config should be required:

```suggestion
function runHook(hook: string, config: MitosisConfig) {
```

Then you'll realize you made a mistake and forgot to provide `config` when you called `runHook` in the code below

Reply:
yes , U are right =_=